chrome-ext/apiHandler.js

```diff
@@ -0,0 +1,15 @@
+class APIHandler {
+    constructor() {
+        this.PICTURE_BASE_URL = "https://www.metaweather.com";
+        this.backgroundApp = axios.create({ baseURL: this.PICTURE_BASE_URL })
+
+    }
+
+    //did not work due to
```

Comment:
If you are not using this part of the code, better comment it

Reply:
true true....